net/vnstat2/files/vnstat.init

```diff
@@ -0,0 +1,60 @@
+#!/bin/sh /etc/rc.common
+
+START=99
```

Comment:
Why 99?
One more service for the "race to be the last one"...

/etc/rc.local should be run last and is started as "95" (as S95done).
Please decrease this to start before that.

Reply:
Thanks for the comment. I didn't really think about this value and just kept the one from the original vnstat package. I'll change it.

Reply:
As it's not a critical service and there are some race conditions I think 99 was chosen for that reason.

Reply:
It looks like the `START` value of 99 for vnStat has been there since 2008: https://git.openwrt.org/?p=openwrt/svn-archive/archive.git;a=blob;f=net/vnstat/files/vnstat.init;h=07db20cba863daca068409a24f03affa12d9952a;hb=b9358dc52de6b8abd99f7b57e18cc68ab4a8b0f9

A race condition is mentioned in https://github.com/openwrt/packages/pull/4682 and https://github.com/openwrt/packages/pull/4615#issuecomment-321765674. Personally, I have never seen this issue (I'm using vnStat with database on a USB stick since 2015). Even `START=60` seems to work for now, but of course I don't have any long-term experience for that.

Another thing I could see is that the system time is more likely to be correct with `START=99` (sysntpd is started as 98). But there is no guarantee for that anyway and this is just a cosmetic issue. And if a persistent database is used vnStat deals with this by itself (by waiting for a configurable time after startup if the system time is before the latest database entry).